index.html

```diff
@@ -56,8 +58,16 @@
         /></a>
       </nav>
     </header>
+    <section class="mobile-menu">
+      <a href="#" class="close-btn" id="close-menu-btn">X</a>
+      <ul class="a-color">
+         <li><h3>Portfolio</h3><a class="listBtns" href="#profile">></a></li>
+         <li><h3>About</h3><a class="listBtns"  href="#about">></a></li>
+         <li><h3>Contact</h3><a class="listBtns"  href="#contact">></a></li>
```

Comment:
- [ ] When the user clicks either of those buttons, a correct part of the page should be displayed
![Screenshot from 2021-10-05 00-40-38](https://user-images.githubusercontent.com/62705947/135934407-fd2a189e-ec4e-4b82-8bb8-8f7222b5f479.png)
d

Reply:
The requirement has been fulfilled 👏🏼